src/Symfony/Component/Validator/Mapping/Factory/LazyLoadingMetadataFactory.php

```diff
@@ -116,9 +116,12 @@ public function getMetadataFor($value)
             $metadata->mergeConstraints($this->getMetadataFor($parent->name));
         }
 
-        // Include constraints from all implemented interfaces
+        // Include constraints from all implemented interfaces that have not been processed via parent class yet
         foreach ($metadata->getReflectionClass()->getInterfaces() as $interface) {
-            if ('Symfony\Component\Validator\GroupSequenceProviderInterface' === $interface->name) {
+            if (
+                'Symfony\Component\Validator\GroupSequenceProviderInterface' === $interface->name ||
+                $parent && $parent->implementsInterface($interface->name)
+            ) {
```

Comment:
Not sure if the codestyle of the if is 100% correct, though fabbot.io is not complaining